plugin.py

```diff
@@ -54,6 +57,18 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self._api = None  # type: Optional[ApiWrapperInterface]
         super().__init__(*args, **kwargs)
 
+    @classmethod
+    def is_allowed_to_start(
+        cls,
+        window: sublime.Window,
+        initiating_view: Optional[sublime.View] = None,
+        workspace_folders: Optional[List[WorkspaceFolder]] = None,
+        configuration: Optional[ClientConfig] = None
+    ) -> Optional[str]:
+        if workspace_folders:
+            register_rename_import_file_watcher(workspace_folders[0].path)
```

Comment:
Should be done for every folder?

Reply:
I plan to add support form multiple folders, 
I just need more time to think about how to handle the cleanup logic (when one folder closes, and so on)

I was thinking of having a Map, where the key is the `root_path` and the value is a `RenameImportHandler` instance.

Reply:
How would you know if the folders have changed?

Reply:
Multiple folders are semi supported.

The current code already supports sublime project.
When having a ST project with multiple folders:
```
// example.sublime-project
{
	"folders":
	[
		{
			"path": "/home/predragnikolic/Documents/sandbox/my-react-ts-app"
		},
		{
			"path": "/home/predragnikolic/Documents/sandbox/coc-tsserver"
		},
		{
			"path": "/home/predragnikolic/Documents/sandbox/volar"
		},
		{
			"path": "/home/predragnikolic/.config/sublime-text/Packages/LSP-typescript"
		}
	]
}

```

It works until the moment you try to add/remove folders to a project while ST is open. :)

Reply:
The current code only takes the path of the first folder.